app/editor/src/features/content/form/ClipDirectoryFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconButton } from 'components/form';
+import React from 'react';
+import { Row, Text } from 'tno-core';
+
+interface IClipDirectoryFilterProps {
+  setGlobalFilter: (filterValue: any) => void;
+}
+
+export const ClipDirectoryFilter: React.FC<IClipDirectoryFilterProps> = ({ setGlobalFilter }) => {
+  const [filter, setFilter] = React.useState<string>('');
+  return (
+    <>
+      <Row className="filter-bar" justifyContent="center">
+        <Text
+          onChange={(e) => {
+            setFilter(e.target.value);
+            setGlobalFilter(e.target.value);
+          }}
+          placeholder="Search by keyword"
+          name="search"
+          value={filter}
+        />
+        <IconButton
+          iconType="reset"
+          onClick={() => {
+            setFilter('');
```

Comment:
I tried eliminating the secondary filter entirely by using global filter; unfortunately, the reset does not work correctly when doing so